hello-world-ruby-webapp/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+displayName: Hello World Web in Ruby
+description: A Hello World web app created using Ruby
+componentType: web-application
+buildPreset: ruby
+repositoryUrl: https://github.com/wso2/choreo-samples/
+componentPath: /hello-world-ruby-webapp
+thumbnailPath: /hello-world-ruby-webapp/icon.png
+documentationPath: /hello-world-ruby-webapp/readme.md
```

Comment:
`readme.md` should be changed to `README.md`. 
Please check readme file path in other places too.

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/15/commits/a67aed655be39d5f54333c8866d56f03c6d43989